pkg/uefi/genericregion.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 the LinuxBoot Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package uefi
+
+// GenericRegion represents the Region in the firmware.
```

Comment:
`// RawRegion implements Region for a raw chunk of bytes in the firmware image.`

Reply:
Done